src/content/blog/2023-01-10-eslint-community-org.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Announcing ESLint's official Community organization
```

Comment:
"ESLint Community" should be capitalized as the name of the organization, but probably not in this line which isn't using the name:

```suggestion
title: Announcing ESLint's official community organization
```

Reply:
Agreed. Maybe better to use that form.
```suggestion
title: Announcing the ESLint Community GitHub organization
```